rigpa.el

```diff
@@ -64,17 +64,12 @@
 (require 'rigpa-history-mode)
 (require 'rigpa-tower-mode)
 
-;; define face for use in meta modes
+;; define face for use in meta modes (make-face 'rigpa-face)
```

Comment:
Did you mean to leave this declaration in the comment?

Reply:
Oops, I did not